sdk/go/common/workspace/plugins.go

```diff
@@ -162,17 +163,20 @@ type MissingError struct {
 	name string
 	// Optional version of the plugin that couldn't be found.
 	version *semver.Version
+	// Optional plugin download url of the plugin that couldn't be found
+	pluginDownloadURL string
```

Comment:
I think at this point `MissingError` should just have a `spec PluginSpec` field or similar.